### ISSUE TITLE
Number serialization does not use neutral culture

### ISSUE THREAD

**Issue by Nils-Berghs** [OfficeDev/Open-XML-SDK#903]
When creating an excel file numbers are serialized with the culture of the current thread. However internally excel uses the invariant/US-culture.

So if you add a value to cell A1 like this:

```
// a value
double value = 0.5;
//assume sheet, row, are created
row.AppendChild(new Cell { CellReference = "A1", CellValue = new CellValue(value), DataType = new EnumValue<CellValues>(CellValues.Number)});
```
Excel will accept the file if the code above is executed in a thread running on US culture. If running on a thread that uses ',' as a decimal seperator, excel will give the "We found a problem witn some content..." warning.
If you examine the value in the zip file it will be "0,5" (not 0.5 as is required).

I have not found a work around other than forcing the the thread to use US culture.

Note that the culture settings of excel do not matter, it is the generation of the xlsx that is wrong.

**Comment by tomjebo** [comment]
This looks like a similar issue as #288 except that our ctor is doing the `ToString()` without doing it like this: `ToString(CultureInfo.InvariantCulture)`
Perhaps we should change the ctors to use the above method? @twsouthwick what do you think?

**Comment by tomjebo** [comment]
@Nils-Berghs I've added PR#904 but don't have a non-US English machine handy. Can you take a look and test your scenario?

**Comment by Nils-Berghs** [comment]
@tomjebo I have tested the branch you are merging and this seems ok.

**Comment by tomjebo** [comment]
@Nils-Berghs thanks for testing. Doing on more confirmation internally and then I'll merge this.

**Comment by tomjebo** [comment]
I've merged #904, added PR #906 for testing double and decimal CellValue. 
Closing this issue.
